.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
You could add:
```yaml
container: docker://l.gcr.io/google/bazel:<some version number>
```
That way the action runs on a container that has Bazel immediately present, meaning that you don't need to install Bazelisk.

Reply:
Those container images seem to lag behind the released bazel versions more than bazelisk, the latest version there is 3.5.0 while the latest bazel release is 3.7.1
https://console.cloud.google.com/gcr/images/cloud-marketplace-containers/GLOBAL/google/bazel?gcrImageListsize=30

Not that it matters too much either way.